common/test/views/support/CommercialTest.scala

```diff
@@ -42,13 +42,4 @@ class CommercialTest extends FlatSpec with Matchers with OptionValues with Befor
   // they should "be responsive for 88x70 ad on US business front" in {
   //   topAboveNavSlot.cssClasses(metaDataFromId("us/business"), defaultEdition, Seq(responsiveSize)) should endWith("top-banner-ad-container--responsive")
   // }
-
-  they should "be default for any other page" in {
```

Comment:
Should this test be removed? If the test was edited to check `endWith("js-top-banner-desktop")`, would it pass?